### ISSUE TITLE
regression: t5717 failing on Windows

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-dev#379]
the same failure occurs in both 2.12.x-integrate-windows and 2.13.x-integrate-windows

seen at e.g. https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-windows/173/consoleFull

```
!! 1190 - run/t5717.scala                           [output differs]
% diff Y:\jenkins\workspace\scala-2.13.x-integrate-windows\test\files\run\t5717-run.log Y:\jenkins\workspace\scala-2.13.x-integrate-windows\test\files\run\t5717.check--- t5717-run.log
+++ t5717.check
@@ -1,1 +1,1 @@
-error: error writing a/B: java.nio.file.FileAlreadyExistsException t5717-run.obj/a
+error: error writing a/B: java.nio.file.FileSystemException t5717-run.obj/a/B.class: Not a directory
##### Log file 'Y:\jenkins\workspace\scala-2.13.x-integrate-windows\test\files\run\t5717-run.log' from failed test #####

error: error writing a/B: java.nio.file.FileAlreadyExistsException t5717-run.obj/a
```

previously: #377

**Comment by lrytz** [comment]
How many times is this one going to hunt me? :-)

**Comment by lrytz** [comment]
https://github.com/scala/scala/pull/5869

**Comment by SethTisue** [comment]
I merged this forward onto 2.13.x too https://github.com/scala/scala/pull/5878
